CSSBot/Services/TheSpookening/spooky.json

```diff
@@ -86,11 +86,19 @@
     "UwU What's this??",
     "P1 Issue at 12 A.M.",
     "git push origin --delete master",
+    "Anish",
+    "pooby's pms",
+    "Aidan Reborn",
+    "garfield without lasagne",
+    "#🅰️nime-quaratine",
+    "​",
```

Comment:
actually hold up, an empty string will just revert the nickname

and zero-width spaces are annoying

Reply:
But invisible people are spooky